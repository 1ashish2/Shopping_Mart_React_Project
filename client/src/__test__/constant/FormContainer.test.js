

import { render, cleanup } from "@testing-library/react";
import FormContainer from "../../constant/FormContainer.constant";
import { MemoryRouter } from 'react-router-dom'



afterEach(() => {
    cleanup();
})

describe("form container Component", () => {
   
    render(<FormContainer  />, { wrapper: MemoryRouter });
    it("matches snapshot", () => {
        const { asFragment } =  render(<FormContainer  />, { wrapper: MemoryRouter });

        expect(asFragment()).toMatchSnapshot();
    })
    
    
    
   
})