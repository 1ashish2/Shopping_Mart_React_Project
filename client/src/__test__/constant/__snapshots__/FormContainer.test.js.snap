// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`form container Component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="form-container container"
  >
    <div
      class="justify-content-md-center  row"
    >
      <div
        class="col-md-6 col-12"
      />
    </div>
  </div>
</DocumentFragment>
`;
